vpr/src/route/sink_sampling.h

```diff
@@ -0,0 +1,155 @@
+#pragma once
+
+/** @file Sink downsamplers for parallel routing. 
+ *
+ * These are used to get a "minimal skeleton routing" from the main task. 
+ * Rest of the routing is delegated to child tasks. They will work with a
+ * strictly limited bounding box, so it's necessary that the initial routing
+ * provides enough hints while routing to as few sinks as possible to limit
+ * the serial bottleneck. */
+
+#include <cmath>
+#include <limits>
+#include <vector>
+#include "globals.h"
+#include "partition_tree.h"
+#include "route_common.h"
+#include "router_lookahead_sampling.h"
+
+/** Sink container for geometry operations */
+struct SinkPoint {
+    int x;
+    int y;
+    int isink;
```

Comment:
Perhaps say what isink represents here, for clarity (the index of the sink among all the sinks of a net)